contrib/mkimage.sh

```diff
@@ -16,16 +16,19 @@ usage() {
 
 scriptDir="$(dirname "$(readlink -f "$BASH_SOURCE")")/mkimage"
 
-optTemp=$(getopt --options '+d:t:h' --longoptions 'dir:,tag:,help' --name "$mkimg" -- "$@")
+optTemp=$(getopt --options '+d:t:c:hC' --longoptions 'dir:,tag:,compression:,no-compression,help' --name "$mkimg" -- "$@")
 eval set -- "$optTemp"
 unset optTemp
 
 dir=
 tag=
+compression="auto"
 while true; do
 	case "$1" in
 		-d|--dir) dir="$2" ; shift 2 ;;
 		-t|--tag) tag="$2" ; shift 2 ;;
+		-c|--compression) compression="$2" ; shift 2 ;;
+		-C|--no-compression) compression="none" ; shift 1 ;;
```

Comment:
I would prefer if we dropped the short form here, especially since `-C` is "change directory" in a lot of other popular tools like `make` and `tar`.